java-checks/src/main/java/org/sonar/java/checks/ReplaceGuavaWithJavaCheck.java

```diff
@@ -33,13 +33,16 @@
 import org.sonar.plugins.java.api.tree.VariableTree;
 
 @Rule(key = "S4738")
-public class ReplaceGuavaWithJava8Check extends AbstractMethodDetection implements JavaVersionAwareVisitor {
+public class ReplaceGuavaWithJavaCheck extends AbstractMethodDetection implements JavaVersionAwareVisitor {
```

Comment:
In addition to renaming this class, we shoud also rename the rule title in the RPSEC:
"Java features should be preferred to Guava"